docs/README.md

```diff
@@ -10,9 +10,9 @@ The editor consists of several primary elements, as shown in the following figur
 
 The elements highlighted in the figure are:
 
-1. **Inserter**: A panel for inserting blocks into the content canvas
-2. **Content canvas**: The content editor, which holds content created with blocks
-3. **Settings sidebar.**: A sidebar panel for configuring a block’s settings (among other things)
+1. **Block Inserter**: A panel for inserting blocks into the content canvas
+2. **Content Canvas**: The content editor, which holds content created with blocks
+3. **Block Inspector (settings sidebar)**: A panel for configuring the selected block’s settings (as a tab of the settings sidebar)
```

Comment:
Same here, I would avoid overusing the term `Block`. The sidebar started as a Block Inspector when there were only blocks in a post, but now it can also display Styles menus, Post settings, etc.